.emacs

```diff
@@ -0,0 +1 @@
+(add-hook 'before-save-hook 'delete-trailing-whitespace)
```

Comment:
Not a big fan of root level configuration files, would love to keep that clean. Can we either move this inside `package.json` or remove it? I don't think many people use emacs with this boilerplate… :grin: